lib/modules/manager/rust/index.ts

```diff
@@ -0,0 +1,14 @@
+import type { Category } from '../../../constants';
+import { GithubReleasesDatasource } from '../../datasource/github-releases';
+import * as rustVersioning from '../../versioning/rust';
+
+export { extractPackageFile } from './extract';
+
+export const supportedDatasources = [GithubReleasesDatasource.id];
+
+export const defaultConfig = {
+  fileMatch: ['(^|/)rust-toolchain\\.toml?$'],
```

Comment:
```suggestion
  fileMatch: ['(^|/)rust-toolchain(\\.toml)?$'],
```

Isn't it the entire .toml extension which should be optional like this?